astro/src/components/ERHeader.astro

```diff
@@ -197,7 +197,6 @@ const breadCrumbs: Breadcrumbs =
                     ><a class="dropdown-item" href="/volunteer">Volunteer</a
                     ></li
                   >
-                  <li><a class="dropdown-item" href="/donate">Donate</a></li>
```

Comment:
I don't see ERHeader being used anywhere but thought I'd remove it here too in case we start using it somewhere.